nupic/research/frameworks/vernon/trainables.py

```diff
@@ -180,33 +274,48 @@ def _train(self):
             self._kill_workers()
             raise
 
-    def _should_stop(self):
+
+class RayActorHelperMethods:
```

Comment:
Could you add a brief docstring to this stating it's used to extend the experiment classes? Or maybe it's obvious enough. Up to you.